caddy/setup/browse.go

```diff
@@ -174,7 +184,7 @@ tr:not(:first-child):hover {
 th,
 td {
 	text-align: left;
-	padding: 10px 0;
+	vertical-align: top;
```

Comment:
Taking out this padding makes it look cramped. I think we should leave the padding there and give the items some room to breath.